pkg/conduit/pipelines.yml

```diff
@@ -0,0 +1,71 @@
+---
+pipelines:
+  pipeline1:
+    status: running
+    config:
+      name: pipeline1
+      description: nothing
+    connectors:
+      con1:
+        type: TYPE_DESTINATION
```

Comment:
Didn't we say we'd use more "human-friendly" values for types?

Reply:
yeah it's just a string.. forgot to change it